### PR TITLE
feat: Add Storybook with example

### PR DESCRIPTION
**Pull request by aaronvanston**
# Overview
- Adds storybook into wingman front-end with example component set-up and structure
- Adds automatic deploy of storybook, unsure how this will play with changeset

### CODE REVIEW

File: .changeset/big-poems-attend.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+'wingman-fe': major
```
Comment (72636c) [inline .changeset/big-poems-attend.md:2]: Could we `minor` this? I'd like to hold on to the v0 for now 😅

File: .changeset/big-poems-attend.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+---
+
+feat: add storybook with Example component structure
```
Comment (72636c) [inline .changeset/big-poems-attend.md:5]: We don't need the semantic prefix here 🙂

File: fe/package.json [hunk 3]
```diff
@@ -12,7 +13,9 @@
     "react-dom": "16.13.1",
     "sku": "10.3.5"
   },
-  "files": ["lib"],
+  "files": [
+    "lib"
+  ],
   "license": "MIT",
   "main": "lib/components/index.ts",
```
Comment (72636c) [inline fe/package.json:20]: Ah, we'll need to repoint this to `lib/index.ts`?

File: .github/workflows/release.yml [hunk 4]
```diff
@@ -35,3 +35,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           NPM_TOKEN: ${{ secrets.SEEK_OSS_CI_NPM_TOKEN }}
+
+      - name: Publish Storybook
+        run: yarn fe storybook:deploy -- --ci --host-token-env-variable=GITHUB_TOKEN
```
Comment (72636c) [inline .github/workflows/release.yml:40]: I assume this will deploy to `/docs` by default?

Comment (aaronvanston) [reply .github/workflows/release.yml:40]: To confirm, it builds into `/dist-storybook` which is then deployed to a `gh-pages` branch.

**Pull request merged**